docs/testing/cart-checkout/readme.md

```diff
@@ -0,0 +1,191 @@
+
+Intermediate:
+- A code editor and/or the ability to modify plugin PHP files.
+  This could be either locally if you're hosting the code there or it could from the Plugins -> Plugin Editor
+  WordPress admin page.
+
+Advanced:
+- A locally installed version of WordPress.
+- [Node 12.16.1 and npm 6.14.4 installed](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/package.json#L149-L150).
+- Ability to edit JS source files when needed.
+
+Special Cases:
+<!-- This section should be moved to stripe payment testing -->
+To test Express payment methods there are some special requirements like
+- You need to be serving the website over https, try ephemeral websites.
+- You need to be from a supported country <!-- Nadir: not yet sure about this condition and what kind of countries are exactly supported or not. -->
+- To test Apple Pay you will need to have an iOS or device.
+- To test Google Pay you will need to have Chrome installed and a payment method setup.
+- To test Microsoft Pay you will need to have Edge installed.
+- You will also need to be on a supported country, to better verify your compatibility visit
+  [this page](https://stripe.com/docs/stripe-js/elements/payment-request-button#react-overview)
+  You should see if you're on a supported platform or not
+
+Unsupported:
+![](https://i.imgur.com/EpkFrat.png)
+
+Supported:
+<!-- someone should screenshot it for me -->
+
+## How to report issues
+
+<!-- We might want to create a more simplified template, that template
+assumes a lot of steps when it comes to accessability. -->
+Ideally, we would prefer if you can submit a issue via [this link](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/new?template=---bug-report.md), however, you can also submit issues here in this thread. <!-- assuming this will be a P2 -->
+
+<!-- This will link to all bugs (open or closed) within a milestone (2.6), if we have a better way to triage bugs, update it here. -->
+It would be preferable to have a look at this [list of issues](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues?q=is%3Aissue+label%3A%22type%3A+bug%22+milestone%3A2.6.0+) to see if the issue you're submitting has already been submitted.
+
+## What are we testing?
+
+The goal is to test the new Cart and Checkout blocks, they should be replacing
+the Cart and Checkout shortcodes.
+
+### Cart Block
+
+![](https://i.imgur.com/mcbXgqV.png)
+
+#### Features to test:
+
+The Cart block has a user story with all [expected features here.](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/1289)
+<!-- debating whether we should be linking to the epic, or copying the epic
+content here, in both cases, the epic should be cleaned and features that are
+not valid for this MVP should be deleted to avoid confusion. -->
+You are also encouraged to compare behaviors with the regular Cart block.
```

Comment:
I think it’s best if we clean and copy past the epic checklist? all of those features cover a good basic testing, maybe some rewording, but I think Pedro has done a good job of making them general enough.

Reply:
Agreed.